python/lsst/sims/GalSimInterface/galSimInterpreter.py

```diff
@@ -571,6 +574,41 @@ def restore_checkpoint(self, camera_wrapper, phot_params, obs_metadata,
             self._rng = image_state['rng']
             self.drawn_objects = image_state['drawn_objects']
 
+    def getHourAngle(self, mjd, ra):
+        """
+        Compute the local hour angle of an object for the specified
+        MJD and RA.
+
+        Parameters
+        ----------
+        mjd: float
+            Modified Julian Date of the observation.
+        ra: float
+            Right Ascension (in degrees) of the object.
+
+        Returns
+        -------
+        float: hour angle in degrees
+        """
+        obs_location = astropy.coordinates.EarthLocation.from_geodetic(
+            self.observatory.getLongitude().asDegrees(),
+            self.observatory.getLatitude().asDegrees(),
+            self.observatory.getElevation())
+        time = astropy.time.Time(mjd, format='mjd', location=obs_location)
+        ra_val = ra if ra > 0 else 360. + ra
+        # Get the local apparent sidereal time.
+        last = time.sidereal_time('apparent').degree
+        last = last if last > 0 else 360. + last
+        ha =  last - ra_val
+        return ha if ha > 0 else 360 + ha
```

Comment:
Also, since you'll eventually want this as a GalSim Angle type anyway, probably best to return this as such, rather than leaving it as a float in degrees.  The point of the Angle class is to make sure you keep track of the units properly, so really best to convert float -> Angle as soon as possible.

Similarly, it's probably bad form to convert the lat/long values from astropy Angles into floats and rely on the default unit being degrees.

So I'd suggest the following for this function
```
    obs_location = astropy.coordinates.EarthLocation.from_geodetic(
           self.observatory.getLongitude(),
           self.observatory.getLatitude(),
           self.observatory.getElevation())
    time = astropy.time.Time(mjd, format='mjd', location=obs_location)
    # Get the local apparent sidereal time.
    last = time.sidereal_time('apparent').degree * galsim.degrees
    ra = ra * galsim.degrees
    ha =  last - ra
    return ha
```